docs/getting-started/styling-messages/markdown.mdx

```diff
@@ -0,0 +1,284 @@
+---
+title: Markdown
+description: Markdown syntax usable to format text on Discord.
+---
+
+import {
+    DiscordInteraction,
+    DiscordMessage,
+    DiscordMessages,
+    DiscordEmbed,
+    DiscordEmbedField,
+    DiscordEmbedFields,
+    DiscordMarkdown,
+} from "discord-message-components/packages/react";
+import "discord-message-components/packages/react/dist/style.css";
+
+import DiscordComponent from "../../../src/components/DiscordComponent";
+
+# Markdown
+Markdown is a type of markup language that's limited in terms of formatting yet simple. Discord allows
+for a watered-down version of markdown to be in their messages.
+
+## Text Markdown
+
+Text formatting can be used in messages and in most embed parts,
+as explained in the dedicated section below.
+ 
+<table>
+    <tr>
+        <td>*italics*</td>
+        <td>__*underlined italics*__</td>
+    </tr>
+    <tr>
+        <td>**bold**</td>
+        <td>__**underlined bold**__</td>
+        
+    </tr>
+    <tr>
+        <td>***bold italics***</td>
+        <td>__***underlined bold italics***__</td>
+        
+    </tr>
+    <tr>
+        <td>__underlined__</td>
+        <td>~~strikethrough~~</td>
+    </tr>
+</table>
+
+## Code Markdown
+
+This formatting can be used not only for code but also for normal text,
+when you want the text to have a different background or to be encased in a box.
+
+### Single-Line Code Block
+
+To create a single-line code block without syntax highlight, wrap the text between single backticks.
+This code block will only add a dark background to the text.
+
+```text
+`print("Hello world!")`
+```
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+`print("Hello world!")`
+    </DiscordMarkdown>
+  </DiscordMessage>
+</DiscordComponent>
+
+<br/>
+
+### Multi-Line Code Block
+
+This type of code block, created by using triple backticks, will encase the text inside a box.
+
+#### Without Syntax Highlight
+
+To create a multi-line code block without syntax highlight, wrap the text between triple backticks
+on first and last line.
+
+````text
+```
+message = "Hello world!"
+print(message)
+```
+````
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+```
+message = "Hello world!"
+print(message)
+```
+    </DiscordMarkdown>
+  </DiscordMessage>
+</DiscordComponent>
+
+<br/>
+
+#### With Syntax Highlight
+
+To create a multi-line block with syntax highlight, add the name of the language you are using right after
+the triple backticks on the first line. For example, for Python you can write either "python" or "py".
+
+````
+```python
+message = "Hello world!"
+print(message)
+```
+````
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+```
```

Comment:
If we want to show this example properly, this code block should be rendered with syntax highlight. Since it was showing "python" as text, I reverted the change for now. The next 2 examples have the same problem.